src/Lumper.UI/ViewModels/MainWindowViewModel.cs

```diff
@@ -1,6 +1,10 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
 using Avalonia;
 using Avalonia.Controls.ApplicationLifetimes;
+using Avalonia.Input;
```

Comment:
Do we need a `using Avalonia` here? I wanted to get them out of the ViewModel at some point. I think the ViewModel souldn't depend on the View.

Reply:
Unfortunately, DragEventArgs relies upon `Avalonia.Input`.

Reply:
yeah didn't find a solution I liked. I _think_ the ViewModel should be reusable even if you change your View framework or whatever. Adding a function in the code-behind which takes the strings and passes them to the ViewModel doesn't sound like the way to go. For unit testing this is probably fine.